src/particles/feedback_CIC_gpu.cu

```diff
@@ -21,7 +25,8 @@
 namespace supernova {
   curandStateMRG32k3a_t*  randStates;
   part_int_t n_states;
-  Real t_buff, dt_buff;
+  Real *d_snr, snr_dt, t_sn_start, t_sn_end;
```

Comment:
Can `snr_dt` be replaced with a `DeviceVector` and reduction like in the hydro time step calculations

Reply:
snr_dt is just a Real number